tanglescope/confirmationratecollector.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <prometheus/exposer.h>
+#include <boost/thread/executor.hpp>
+#include <boost/thread/future.hpp>
+#include <chrono>
+#include <iota/tanglescope/common/iri.hpp>
+#include <libcuckoo/cuckoohash_map.hh>
+#include <list>
+#include <map>
+#include <memory>
+#include <rx.hpp>
+#include <set>
+#include <string>
+
+#include "tanglescope/throwcatchcollector.hpp"
+
+#include "cppclient/beast.h"
+
+namespace iota {
+namespace tanglescope {
+
+class CRCollector : public ThrowCatchCollector {
```

Comment:
could you add documentation of what this class does? also maybe some algo documentation to the methods?